core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -67,6 +67,12 @@ trait Fiber[+E, +A] { self =>
    */
   def interrupt: UIO[Exit[E, A]]
 
+  /**
+   * Inherits values from all [[FiberRef]] instances into current fiber.
+   * This will resume immediately.
+   */
+  def inheritLocals: UIO[Unit]
```

Comment:
Rename to `inheritFiberRefs` as discussed, or maybe just, `inheritRefs`.

Reply:
done